### PR TITLE
Assume the maven wrapper is in the directory with the build file pom.xml

### LINKED ISSUE #563: FVT: Opening a parent folder with two Maven projects doesn't trigger the "Start action."
In Linux, when you try to open a parent folder containing two Maven projects, the "Start action" is not triggered.

I tried Using the below Configurations
OS : RedHat Linux 
Version : 9.2
Intellij Version: 2023.2.4

Screenshot 2023-11-03 at 3 26 33 PM
Screenshot 2023-11-06 at 11 45 50 AM

### PR DESCRIPTION
**Pull request by turkeylurkey**
Fixes #563

~Also, if the maven wrapper near the build file fails the checks, try to use the maven wrapper at the project level.~

### CODE REVIEW

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java [hunk 1]
```diff
@@ -214,24 +216,35 @@ public static String getMavenSettingsCmd(Project project) throws LibertyExceptio
...
-        if (!file.canExecute()) {
+        if (!wrapper.canExecute()) {
             String translatedMessage = LocalizedResourceUtil.getMessage("maven.wrapper.cannot.execute");
             throw new LibertyException("Could not execute Maven wrapper because the process does not have permission to " +
                     "execute it. Consider giving executable permission for the Maven wrapper file or changing the build " +
                     "preferences for Maven inside IntelliJ Maven preferences.", translatedMessage);
         }
+        if (bfWrapper != null) {
+            LOGGER.warn("The wrapper built from the build file is invalid, using wrapper from project directory. exists="
+                    + bfWrapper.exists() + " canExecute=" + bfWrapper.canExecute());
```
Comment (TrevCraw) [inline src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java:245]:
```suggestion
            LOGGER.warn("A wrapper in the build file directory is either invalid or missing. Using wrapper from project base directory. Build file wrapper values: exists="
                    + bfWrapper.exists() + " canExecute=" + bfWrapper.canExecute());
```

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java [hunk 2]
```diff
@@ -214,24 +216,35 @@ public static String getMavenSettingsCmd(Project project) throws LibertyExceptio
...
         }
-        if (!file.canExecute()) {
+        if (!wrapper.canExecute()) {
             String translatedMessage = LocalizedResourceUtil.getMessage("maven.wrapper.cannot.execute");
             throw new LibertyException("Could not execute Maven wrapper because the process does not have permission to " +
                     "execute it. Consider giving executable permission for the Maven wrapper file or changing the build " +
                     "preferences for Maven inside IntelliJ Maven preferences.", translatedMessage);
         }
+        if (bfWrapper != null) {
+            LOGGER.warn("The wrapper built from the build file is invalid, using wrapper from project directory. exists="
```
Comment (TrevCraw) [inline src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java:244]: should this be a warning? I think `info` or `debug` would be more appropriate

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java (hunk 1 above)
Comment (turkeylurkey) [reply src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java]: Updated this message in the alternate branch where this code is stored for future consideration.

File: src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java (hunk 2 above)
Comment (turkeylurkey) [reply src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java]: I thought the IntelliJ log is normally not visible? Also info messages do not appear in the log unless you modify the settings. I moved this code into another branch for future consideration.

**Review by dessina-devasia — APPROVED** [review]

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**